### ISSUE TITLE
Missing No

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1026]
would be actually takeable for T lost if it gave him holy mantle (if he doesn't have one) every floor (just like it would be nice for Esau Jr to give mantle)

**Comment by Turtier** [comment]
I'm assuming missingno on tlost specifically is only a problem because his "better items" mechanic doesn't work on d4 effects? If so, the only buff that would make sense would be for his mechanic to apply to d4. Either way, missingno is actually takeable if you abuse d4 bug to turn most your rerolls into devil items (pretty sure this bug is reasonably well known and is intentionally not being patched, which is great imo)

**Comment by Lonrite** [comment]
yeah I suppose although holy cards would also be neat to have lol

**Comment by Turtier** [comment]
Its just that missingno giving you a holy card doesn't make any sense. Like changing tlost esau jr to have a holy card kinda makes sense because if you have one used on tlost already, it'll go to waste, but missingno doesn't take away your holy cards or your mantle, so it doesn't make sense for it to compensate you with holy cards that it never took away to begin with

**Comment by Lonrite** [comment]
i guess lol, it still would be a nice bonus imo even if it doesn't make sense since Missing No is garbage tier but I guess same can be said about TMTRAINER and it giving a holy card is reaching too far lol
